### PR TITLE
Add 404 page

### PR DESCRIPTION
**Pull request by dev-rminds**

### CODE REVIEW

File: src/sources/forus-webshop/js/angular-1/i18n/nl/pages/error-page.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    // error-page.pug
+    404: {
+        title: 'Page not found',
```
Comment (maxvisser) [inline src/sources/forus-webshop/js/angular-1/i18n/nl/pages/error-page.js]: Pagina niet gevonden

**Comment by maxvisser** [comment]
image
Seems to be english? Not sure why

**Comment by maxvisser** [comment]
Back to home button does not seem to work

**Review by maxvisser — CHANGES_REQUESTED** [review]
- Page should be dutch
- Back to home button should return user to home.

**Comment by maxvisser** [comment]
Not sure if this will work on staging. Needs testing there before merging.

**Review by GerbenBosschieter — APPROVED** [review]

**Pull request merged**